### PR TITLE
docs(xlm): example transfer

### PR DESCRIPTION
**Pull request by marianogoldman**
## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

**Comment by faustbrian** [comment]
All of the examples will probably need adjusting after the signatory changes.

File: examples/bin/run [hunk 1]
```diff
@@ -29,3 +30,7 @@ if (input[0] === "transfer-ark") {
 if (input[0] === "transfer-trx") {
 	(async () => transferTrx())();
 }
+
+if (input[0] === "transfer-xlm") {
+	(async () => transferXlm())();
```
Comment (axeldelamarre) [inline examples/bin/run:35]: Also possible to wraps the whole file in
```typescript
(async () => {
  //
})();
to avoid repeating the aync

Comment (marianogoldman) [reply examples/bin/run:35]: Awesome! Thanks @axeldelamarre.

**Comment by marianogoldman** [comment]
> All of the examples will probably need adjusting after the signatory changes.

Sure @faustbrian. That's why it would be better if it was a project that had tests and everyone would notice and fix following code changes. Not meant to be a publishable module, but is there any way we can execute these with nock mocks during CI?

**Comment by faustbrian** [comment]
Maybe we could make these commands part of the `platform-sdk-cli` package? Could prefix those commands with `test:`.

**Comment by faustbrian** [comment]
Yeah but that will be easier once we integrate them into a proper package. I can prepare a PR tomorrow.

**Pull request merged**